Public/Src/IDE/VsCode/client/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "buildxldscript",
+  "name": "dscript",
```

Comment:
>dscript [](start = 11, length = 7)

It turns out that the vscode and the publish scripts look at this name as the identity, but the backend marketplace looks at the extension.vsixmanifest 'id' field in [https://github.com/microsoft/BuildXL/blob/master/Public/Src/IDE/VsCode/pluginTemplate/extension.vsixmanifest](https://github.com/microsoft/BuildXL/blob/master/Public/Src/IDE/VsCode/pluginTemplate/extension.vsixmanifest)

Since that one was already baked into the market place I opted to use that id here rather than update the old, we'd have to ask everybody to uninstall the plugin and then reinstall the plugin again...